lib/foreman/gettext/all_domains.rb

```diff
@@ -1,23 +1,29 @@
-require 'fast_gettext'
-
 # include this module to translate in all domains by default
 module Foreman
   module Gettext
     module AllDomains
+      class Localizator
+        prepend FastGettext::TranslationMultidomain
```

Comment:
I tried to use `include` here and then it crashes. I also had the impression this file was loaded twice. At least `debug` appears to be:
```
/home/ekohl/dev/foreman/lib/foreman/gettext/debug.rb:7: warning: already initialized constant Foreman::Gettext::Debug::DL
/home/ekohl/dev/foreman/lib/foreman/gettext/debug.rb:7: warning: previous definition of DL was here
/home/ekohl/dev/foreman/lib/foreman/gettext/debug.rb:8: warning: already initialized constant Foreman::Gettext::Debug::DR
/home/ekohl/dev/foreman/lib/foreman/gettext/debug.rb:8: warning: previous definition of DR was here
```

So perhaps that's related to the infinite recursion.

Reply:
Well, when I was checking (not `debug` version), I've noticed that due to changes in the library, we ended up calling our `_()` method instead of the lib's one due to `including`  stuff wrongly. I don't think double loading has something to do with the recursion. Might be wrong though...